### PR TITLE
[Card-14] Create README.md files

### PR DESCRIPTION
**Pull request by minhajdevcrew**

### CODE REVIEW

**Comment by minhajdevcrew** [comment]
[Publish Package "Expandable Rich Text"](https://trello.com/c/lV82Ctsh/14-publish-package-expandable-rich-text)

**Review by WaseemAbbas-DevCrew — COMMENTED** [review]
@taimoor-dc please review all doc files. Thank you.

**Comment by minhajdevcrew** [comment]
- [README.md] https://github.com/DevCrew-io/expandable-richtext/tree/card-14-publish-package
- [example/README.md] https://github.com/DevCrew-io/expandable-richtext/tree/card-14-publish-package/example
- [CHANGELOG.md] https://github.com/DevCrew-io/expandable-richtext/blob/card-14-publish-package/CHANGELOG.md
- [LICENSE] https://github.com/DevCrew-io/expandable-richtext/blob/card-14-publish-package/LICENSE

@WaseemAbbas-DevCrew @taimoor-dc

**Review by taimoor-dc — APPROVED** [review]
The LICENSE file seems okay and is good to go.

**Review by taimoor-dc — APPROVED** [review]
The README file is approved for merging.

**Review by taimoor-dc — APPROVED** [review]
The CHANGELOG file seems okay.

File: README.md [hunk 1]
```diff
@@ -1,39 +1,111 @@
...
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages).
--->
-
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+[![pub package](https://img.shields.io/pub/v/expandable_text.svg)](https://pub.dev/packages/expandable_rich_text)
+[![license](https://img.shields.io/badge/license-MIT-green)](https://github.com/DevCrew-io/expandable-richtext/blob/main/LICENSE)
+A text widget which you can use to initially only show a defined number of lines of a probably long text. it also support clickable @‌mention, #hashtag and hyperlinks. You can define styles for @‌mention, #hashtag and hyperlinks, also apply custom style on segments of text.
```
Comment (taimoor-dc) [inline README.md]: A text widget that you can use to initially only show a defined number of lines of a probably long text. It also supports clickable @‌mention, #hashtag, and hyperlinks. You can define styles for @‌mention, #hashtag, and hyperlinks and apply custom styles on text segments.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,111 @@
...
 
 ## Features
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+- Collapse and Expand text
+- Configure text to show to Expand text (`expandText`) and to Collapse text (`collapseText`)
+- Control whether the ellipsis (`showEllipsis`) is part of the Expand/Collapse text
+- Expande and Collapse animation
```
Comment (taimoor-dc) [inline README.md]: Expand

File: README.md [hunk 3]
```diff
@@ -1,39 +1,111 @@
...
+
+Next, import the package into your dart code:
+
+```dart
+import 'package:expandable_rich_text/expandable_rich_text.dart';
+```
+
+### Usage
+
+If `longText` exceeds one two:
```
Comment (taimoor-dc) [inline README.md]: exceeds one two...??

**Review by taimoor-dc — CHANGES_REQUESTED** [review]
Single-line comments added on README file that need to be addressed before merging.

File: README.md (hunk 1 above)
Comment (minhajdevcrew) [reply README.md]: sure thanks. I will update this description.

File: README.md (hunk 2 above)
Comment (minhajdevcrew) [reply README.md]: 'Expand' i did not get this.

File: README.md (hunk 3 above)
Comment (minhajdevcrew) [reply README.md]: sure thanks.  I will fix this.

File: README.md (hunk 2 above)
Comment (taimoor-dc) [reply README.md]: You need to correct the spellings.

File: README.md (hunk 3 above)
Comment (minhajdevcrew) [reply README.md]: fix: If `longText` exceeds two lines:

**Review by AliRehan-DevCrew — APPROVED** [review]

**Review by taimoor-dc — APPROVED** [review]
Approved for merging.

**Pull request merged**